portfolio/src/main/java/com/google/sps/servlets/Comments.java

```diff
@@ -29,10 +29,9 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
+@WebServlet("/comments")
```

Comment:
Add a java doc for the class. A comment describing what the class does
/** Servelet for handling comments posted by the user. */

Reply:
Done: 29fdbd4